PokemonRampUp/Model/PokemonJSONToPokemon.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  PokemonJSONToPokemon.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-03-02.
+//
+
+import Foundation
+
+// TO-DO [POKEMON-0014] Create unit test for PokemonJSONToPokemon
+class PokemonJSONToPokemon{
+    
+    let pokemonJSON: PokemonJSON
+    let images: Images
+    
+    init(pokemonJSON: PokemonJSON, images: Images) {
+        self.pokemonJSON = pokemonJSON
+        self.images = images
+    }
+    
+    private func capitalizeFirstLetterOfNameContainers(nameContainers: [NameContainerProtocol]) -> [NameContainerProtocol]{
```

Comment:
I made `Ability`, `TypeElement` and `Move` structs conform to NameContainerProtocol. By doing this I would be able to upcast those structs so I would not have to make 3 separate functions